tiledb/sm/filter/CMakeLists.txt

```diff
@@ -192,9 +194,10 @@ target_link_libraries(filter_pipeline PUBLIC tile $<TARGET_OBJECTS:tile>)
 # Test-compile of object library ensures link-completeness
 #
 add_executable(compile_filter_pipeline EXCLUDE_FROM_ALL)
+add_dependencies(all_link_complete compile_filter_pipeline)
 target_link_libraries(compile_filter_pipeline PRIVATE filter_pipeline)
 target_sources(compile_filter_pipeline PRIVATE test/compile_filter_pipeline_main.cc)
-
+add_dependencies(all_link_complete compile_filter_pipeline)
```

Comment:
Are both instances of `add_dependencies(all_link_complete compile_filter_pipeline)` necessary?

Reply:
Removed, thanks.